packages/yoroi-extension/app/ergo-connector/stores/ConnectorStore.js

```diff
@@ -232,23 +232,24 @@ export default class ConnectorStore extends Store<StoresMap, ActionsMap> {
     try {
       const wallets = await getWallets({ db: persistentDb });
 
-      const ergoWallets = wallets.filter(
-        wallet => isErgo(wallet.getParent().getNetworkInfo())
-      );
+      const filteredWallets = wallets
+      // list all wallet for now until we find a way to
+      // know which type of wallet to filter ( ergo || cardano )
+      // .filter(wallet => !isErgo(wallet.getParent().getNetworkInfo()));
 
-      await this._getTxAssets(ergoWallets);
+      await this._getTxAssets(filteredWallets);
```

Comment:
First, when you request a connection to Yoroi it displays  by default the ergo wallets 
Now we want it to display Ergo or Cardano  wallets according to the connection request